### PR TITLE
typo combing

### PR DESCRIPTION
**Pull request by Round-Pi**
# What's new

- fix typos, with the help of the spell check extension on VS Code

# Verification 
- Build fbt
- Update firmware on Flipper Zero and see if anything breaks

# Checklist (For Reviewer)

- [x] PR has description of feature/bug
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

**Review by xMasterX — CHANGES_REQUESTED** [review]
Hi, amiibo was named ameebo to avoid nintendo copyrights issues
Do not rename that

Second thing, you changed submodules
lib/FreeRTOS-Kernel
lib/STM32CubeWB
We can't accept such changes

also revert changes to debug/fw.jflash

**Comment by Round-Pi** [comment]
Alright, I made the changes.

File: firmware/targets/f7/ble_glue/app_conf.h [hunk 1]
```diff
@@ -463,4 +463,4 @@ typedef enum {
 #define MAX_DBG_TRACE_MSG_SIZE 1024
 
 #define CFG_OTP_BASE_ADDRESS OTP_AREA_BASE
-#define CFG_OTP_END_ADRESS OTP_AREA_END_ADDR
```
Comment (xMasterX) [inline firmware/targets/f7/ble_glue/app_conf.h]: This needs to be reverted

also Ameebo needs to be reverted

**Review by xMasterX — CHANGES_REQUESTED** [review]

**Comment by Round-Pi** [comment]
Done. Are there any other changes I should fix?

**Comment by xMasterX** [comment]
> Done. Are there any other changes I should fix?

thanks, other things looks ok, i will test that nothing is broken today/tomorrow and will merge it

**Comment by Round-Pi** [comment]
If it doesn't work, I can revert all the typo-fixes, except for the ones in comments and documents.

**Review by xMasterX — APPROVED** [review]
Thanks!

**Pull request merged**